grafonnet/text.libsonnet

```
@@ -10,6 +23,9 @@
     transparent=null,
     description=null,
     datasource=null,
+    repeat=null,
+    repeatDirection='h',
```

Grafana will use 'h' as the default so there's no need to set this unless explicitly done so.

```suggestion
    repeatDirection=null,
```